jboss-kie-wildfly-common/added/launch/jboss-kie-wildfly-security-login-modules.sh

```diff
@@ -71,7 +76,7 @@ function configure_ldap_login_module() {
     # RHPAM-1422, if the RealmDirect is set as Required, ldap auth will fail.
     # TODO remove it out as part of the CLOUD-2750
     sed -i 's|<login-module code="RealmDirect" flag="required">|<login-module code="RealmDirect" flag="optional">|' "${CONFIG_FILE}"
-    local login_module=$(build_login_module "LdapExtended" "required")
+    local login_module=$(build_login_module "LdapExtended" "${AUTH_LDAP_FLAG:-required}")
```

Comment:
what happens if use sets the flag to a non sense value?
we need to make sure that the value configured by user is valid;